baselines/dom.generated.d.ts

```diff
@@ -7112,7 +7112,7 @@ interface IDBRequestEventMap {
 }
 
 interface IDBRequest extends EventTarget {
-    readonly error: DOMError;
+    readonly error: DOMException | DOMError;
```

Comment:
I would say `DOMException`. from usage perspective `DOMException` has the same props as `DOMError`, so it is not going to break existing code. but it allows for users using new browsers to use the new APIs.